### PR TITLE
Netherlands Translations

### PR DESCRIPTION
**Pull request by leigh-pointer**
*.nl-NL.resx files created

### CODE REVIEW

File: English (en)/Oqtane.Client/Resources/Modules/Admin/Login/Index.resx [hunk 1]
```diff
@@ -126,4 +126,13 @@
   <data name="Forgot Password" xml:space="preserve">
     <value>Forgot Password</value>
   </data>
+  <data name="Password:" xml:space="preserve">
```
Comment (hishamco) [inline English (en)/Oqtane.Client/Resources/Modules/Admin/Login/Index.resx:129]: Did you run the latest version of the resource extractor? I see some changes not related to `nl-NL`

Comment (leigh-pointer) [reply English (en)/Oqtane.Client/Resources/Modules/Admin/Login/Index.resx:129]: Commit on Apr 6, 2021 this is fixed

Comment (hishamco) [reply English (en)/Oqtane.Client/Resources/Modules/Admin/Login/Index.resx:129]: I don't understand why the change happen on non `nl-NL`?

Comment (hishamco) [reply English (en)/Oqtane.Client/Resources/Modules/Admin/Login/Index.resx:129]: I merge you changes, I will try to refresh the English resources later

Thanks a lot @leigh-pointer

**Pull request merged**